resources/packs/gcp/logging.go

```diff
@@ -0,0 +1,116 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	"go.mondoo.com/cnquery/resources"
+	"go.mondoo.com/cnquery/resources/packs/core"
+	"google.golang.org/api/logging/v2"
+	"google.golang.org/api/option"
+)
+
+func (g *mqlGcpProjectLoggingservices) id() (string, error) {
+	return "gcp.project.loggingservices", nil
```

Comment:
This will not work for multiple instances for multiple projects since they return always the same id.

```
func (g *mqlGcpProjectLoggingservices) id() (string, error) {
  projectId, err := g.ProjectId()
  if err != nil { return "", err}
  
  return "gcp.project.loggingservices/"+ projectID, nil
}
```